### DISCUSSION TITLE
ch_PP-OCRv4_det_cml.yml里面的PPLCNetV3的模型文件和开源的ch_PP-OCRv4_det模型对应不上

### DISCUSSION THREAD

**Discussion by wuxiaolianggit** [PaddlePaddle/PaddleOCR#12921] [Q&A]
@sdcb @ZeyuChen @haobibo @bingooo @llxxxll 大佬们，您好啊，ch_PP-OCRv4_det_cml.yml里面的PPLCNetV3的模型文件和开源的ch_PP-OCRv4_det模型对应不上，模型参数里面没有se模块，但是模型文件RSEFPN有se模块，把这个模块去掉之后，把paddle转成pth还是会掉精度，不知道怎么回事呢?大佬你有真正的ch_PP-OCRv4_det的模型定义文件吗？

**Comment by frotms** [comment]
train模型权重里确实没有neck的se模块的参数，转inference模型后由于缺se模块参数，和提供下载的inference模型结果也不一致
